tests/jenkins/TestDataPrepperDistributionArtifacts.groovy

```diff
@@ -3,35 +3,55 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import jenkins.tests.BaseJobPipelineTest
+
+import jenkins.tests.BuildPipelineTest
 import org.junit.Before
 import org.junit.Test
 
-class TestDataPrepperDistributionArtifacts extends BaseJobPipelineTest {
+class TestDataPrepperDistributionArtifacts extends BuildPipelineTest {
 
     @Before
     void setUp() {
         super.setUp()
 
+        // Variables for Data-prepper-Distribution-Artifacts-job
+        def workspace = '/tmp/workspace'
         binding.setVariable('VERSION', '0.22.1')
         binding.setVariable('BRANCH', 'refs/tags/0.22.1')
-
         binding.setVariable('JOB_BASE_NAME', 'data-prepper-distribution-artifacts')
-        binding.setVariable('BUILD_NUMBER', '7654321')
-        binding.setVariable('WORKSPACE', '/tmp/workspace')
-
+        binding.setVariable('BUILD_NUMBER', '51')
+        binding.setVariable('WORKSPACE', workspace)
         binding.setVariable('ARTIFACT_BUCKET_NAME', 'job-s3-bucket-name')
 
         helper.registerAllowedMethod('checkout', [Map], {})
-        helper.registerAllowedMethod('signArtifacts', [Map], {})
-        helper.registerAllowedMethod('uploadToS3', [Map], {})
-        helper.registerAllowedMethod('printArtifactDownloadUrlsForStaging', [Map], {})
-        helper.registerAllowedMethod('postCleanup', [], {})
+
+        def artifactsPath = "$workspace/release/archives/linux/build/distributions"
+        helper.addShMock("find ${artifactsPath} | sed -n \"s|^${artifactsPath}/||p\"") { script ->
+            return [stdout: "tar_dummy_1_artifact_1.0.0.tar.gz tar_dummy_2_artifact_1.0.0.tar.gz", exitValue: 0]
```

Comment:
These strings don't mean anything to the Data Prepper job. What are they? And how does a job author manage them?